### PR TITLE
Lab Users' Contributed Lab Repos

### PR DESCRIPTION
**Pull request by LRWeber**
Added new data collection, newly generated versions of labUsers.json should now mirror extUsers.json by including a list of lab repositories each user has contributed to.

### CODE REVIEW

File: explore/scripts/get_repos_users.py [hunk 1]
```diff
@@ -31,7 +33,7 @@
 
 # Iterate through internal repos
 print("Gathering data across multiple paginated queries...")
-collective = {u'data': {}}
+collectiveExt = {u'data': {}}
```
Comment (IanLee1521) [inline explore/scripts/get_repos_users.py]: Do you need a setup of `collectiveInt` here? I don't see the creation of that objection anywhere... Just usage down in line 65, which is going to break I suspect.

File: explore/scripts/get_repos_users.py [hunk 2]
```diff
@@ -52,19 +54,23 @@
 	outObj = helpers.query_github(authhead, gitquery)
 	if outObj["errors"]:
 		print(tab + "Could not complete '%s'" % (repo))
-		collective["data"].pop(repo, None)
+		collectiveExt["data"].pop(repo, None)
```
Comment (IanLee1521) [inline explore/scripts/get_repos_users.py]: Same here.

File: explore/scripts/get_repos_users.py [hunk 3]
```diff
@@ -52,19 +54,23 @@
...
 		userKey = user["login"]
 		if userKey in memberlist:
-			continue
-		if userKey not in collective["data"].keys():
-			collective["data"][userKey] = user
-			collective["data"][userKey]["contributedLabRepositories"] = {"nodes": []}
-		collective["data"][userKey]["contributedLabRepositories"]["nodes"].append(repo)
-		collective["data"][userKey]["contributedLabRepositories"]["nodes"].sort()
+			if "contributedLabRepositories" not in collectiveInt["data"][userKey].keys():
+				collectiveInt["data"][userKey]["contributedLabRepositories"] = {"nodes": []}
```
Comment (IanLee1521) [inline explore/scripts/get_repos_users.py]: Accessed here.

File: explore/scripts/get_repos_users.py [hunk 4]
```diff
@@ -103,12 +113,22 @@
...
+
+# Combine new data with existing data
+allData["data"] = collectiveInt["data"]
 allDataString = json.dumps(allData, indent=4, sort_keys=True)
 
 # Write output file
-print("\nWriting file '%s'" % (datfilepath))
-with open(datfilepath, "w") as fileout:
+print("\nWriting file '%s'" % (datfilepathInt))
+with open(datfilepathInt, "w") as fileout:
```
Comment (IanLee1521) [inline explore/scripts/get_repos_users.py]: Do you need two data writes here? One for int and one for ext?

File: explore/scripts/get_repos_users.py (hunk 1 above)
Comment (LRWeber) [reply explore/scripts/get_repos_users.py]: Line 20.
Already ran a full update #80 , no breakage.

File: explore/scripts/get_repos_users.py (hunk 4 above)
Comment (LRWeber) [reply explore/scripts/get_repos_users.py]: Yup, two different files, two data writes.
datfilepathExt = "../github-data/extUsers.json" (lines 115-123)
datfilepathInt = "../github-data/labUsers.json" (lines 125-133)

**Comment by IanLee1521** [comment]
Doh! I missed line 20... I figured it was working, as I did see your other PR, sorry for the noise!

**Pull request merged**